crates/tarball/Cargo.toml

```
@@ -28,6 +28,7 @@ tar          = { workspace = true }
 tokio        = { workspace = true }
 zune-inflate = { workspace = true }
 tracing      = { workspace = true }
+rayon.workspace = true
```

```suggestion
rayon = { workspace = true }
```

(remember to align the columns, or just run `taplo format`)